src/FsCheck.NUnit/paket.template

```diff
@@ -1,31 +0,0 @@
-type file
-id
-    FsCheck.NUnit
-authors
-    Kurt Schelfthout and contributors
-owners
-    Kurt Schelfthout and contributors
-projectUrl
-    https://github.com/fsharp/FsCheck
-iconUrl
-    https://raw.githubusercontent.com/fscheck/FsCheck/master/docs/files/img/logo.png
-licenseUrl
```

Comment:
This is deprecated; it's already implied by the PackageLicenseExpression in Directory.Build.props.